### PR TITLE
Lesson 1.7

### PR DESCRIPTION
**Pull request by Jay-zar**
The To Do List does not persist on refresh now. I think if I understood the instructions for this and next week, that is correct for now and we will add a call to a database next week. Sorry if that's incorrect.

### CODE REVIEW

File: src/InputWithLabel.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React, { useRef, useEffect } from 'react';
+
+const InputWithLabel= ({children, toDoTitle, handleTitleChange}) => {
+    const inputRef= React.useRef();
```
Comment (jkylehumphrey) [inline src/InputWithLabel.js:4]: since you've imported `useRef` no need to qualify it with `React.`

File: src/App.js [hunk 2]
```diff
@@ -1,19 +1,37 @@
...
+      setToDoList(result.data.toDoList);
+      setIsLoading(false);
+    })
+  }, []);
+  //setting list from the input box
+  useEffect(()=> {
+    if (isLoading === false) {
+      localStorage.setItem('savedToDoList', JSON.stringify(toDoList))
+    }
+  }, [toDoList]);
```
Comment (jkylehumphrey) [inline src/App.js:24]: I've asked about this with the mentors and havne't gotten an answer but: technically, this hook should also depend on `isLoading` b/c it's used in the hook.  It works fine as it is and the lesson said do it this way, so it's fine.  Just thought I'd leave a note.

**Review by jkylehumphrey — COMMENTED** [review]
Looks good!  Left a few comments in the code

File: src/App.js (hunk 2 above)
Comment (Jay-zar) [reply src/App.js:24]: Ah thank you, I was a little confused about that.

**Pull request merged**